ajax/libs/native-promise-only/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "native-promise-only",
+  "filename": "npo.js",
+  "author": {
+    "email": "getify@gmail.com",
+    "name": "Kyle Simpson"
+  },
```

Comment:
@nolsherry You don't need to seperate `"author": "Kyle Simpson <getify@gmail.com>"` It;s better to be same as upstream.